resources/wikia/modules/geo.js

```diff
@@ -164,11 +172,11 @@
 	//TODO: Can we remove the double alias in window.Geo and Wikia.geo
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)